golangci-lint error on master branch

I'm seeing a lint error on master when running the full `make`. Not sure if this started recently or has been there for a while but I can repro on `commit 3aeb251` 

```
Running golangci-lint
golangci-lint run ./...
build/manifest/main.go:106:13: G306: Expect WriteFile permissions to be 0600 or less (gosec)
		if err := ioutil.WriteFile(
			"server/manifest.go",
			[]byte(fmt.Sprintf(pluginIDGoFileTemplate, manifest.Id, manifest.Version)),
			0644,
		); err != nil {
build/manifest/main.go:116:13: G306: Expect WriteFile permissions to be 0600 or less (gosec)
		if err := ioutil.WriteFile(
			"webapp/src/manifest.js",
			[]byte(fmt.Sprintf(pluginIDJSFileTemplate, manifest.Id, manifest.Version)),
			0644,
		); err != nil {
make: *** [golangci-lint] Error 1
```

--- a/build/manifest/main.go
+++ b/build/manifest/main.go
@@ -106,7 +106,7 @@ func applyManifest(manifest *model.Manifest) error {
 		if err := ioutil.WriteFile(
 			"server/manifest.go",
 			[]byte(fmt.Sprintf(pluginIDGoFileTemplate, manifest.Id, manifest.Version)),
-			0644,
+			0600,
 		); err != nil {
 			return errors.Wrap(err, "failed to write server/manifest.go")
 		}
@@ -116,7 +116,7 @@ func applyManifest(manifest *model.Manifest) error {
 		if err := ioutil.WriteFile(
 			"webapp/src/manifest.js",
 			[]byte(fmt.Sprintf(pluginIDJSFileTemplate, manifest.Id, manifest.Version)),
-			0644,
+			0600,
 		); err != nil {
 			return errors.Wrap(err, "failed to open webapp/src/manifest.js")
 		}

